Makefile

```diff
@@ -13,7 +13,7 @@ objects		= $(HTTP_PARSER_OBJ) \
 		             $(wildcard $(SOURCE_DIR)/*.c))
 
 CPPFLAGS	+= $(PYTHON_INCLUDE) -I . -I $(SOURCE_DIR) -I $(HTTP_PARSER_DIR)
-CFLAGS		+= $(FEATURES) -std=c99 -fno-strict-aliasing -fcommon -fPIC -Wall
+CFLAGS		+= $(FEATURES) -std=c11 -fno-strict-aliasing -fcommon -fPIC -Wall
```

Comment:
I never get the Makefile working for me, but I did need to change to c11 in the setup.py otherwise I face an issue with the struct timespec not defined issue.